source/visionEnhancementProviders/exampleProvider_autoGui.py

```diff
@@ -0,0 +1,188 @@
+# A part of NonVisual Desktop Access (NVDA)

```

Comment:
What are you intentions with this example? I assume it won't end up in the real pr? NOte that if you prefix the module name with an underscore, it will not be ignored by the framework.

Reply:
> What are you intentions with this example?

At first I planned to just past it into the `readme.md` as example code. But I am worried it will become unmaintained, especially while this subsystem is new and prone to changes. At the moment I'm considering whether it can stay where it is and either return `False` from `canStart` or renamed to start with an underscore, if that stops it from being picked up. I want it to be easy to test with, since we don't have a built-in provider using the AutoGui panels.

>  it will not be ignored by the framework.

Do you mean it WILL be ignored?